rosdep/python.yaml

```diff
@@ -5104,6 +5104,23 @@ python3-bson:
       packages: [bson]
   rhel: ['python%{python3_pkgversion}-bson']
   ubuntu: [python3-bson]
+python3-build:
+  alpine: [py3-build]
+  arch: [python-build]
+  debian: [python3-build]
+  fedora: [python3-build]
+  gentoo: [dev-python/build]
+  osx: [python-build]
+  rhel:
+    rhel8:
+      pip:
+        packages: [build]
+    rhel9: [python3-build]
+  ubuntu:
+    focal:
+      pip:
+        packages: [build]
```

Comment:
Unless you specifically need the pip packages on Focal it's better not to add them.

```suggestion
```